app/scripts/modules/core/src/pipeline/config/stages/googleCloudBuild/GoogleCloudBuildStageForm.tsx

```diff
@@ -15,16 +15,21 @@ import {
   StageConfigField,
   yamlDocumentsToString,
   YamlEditor,
+  TextInput,
 } from '@spinnaker/core';
+import { IGcbTrigger } from 'core/domain';
```

Comment:
Same comment as above -- let's add this import to the existing `@spinnaker/core` import.